### PR TITLE
Minor tips update

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Updates some game tips in various ways, mainly just rewording a few of them to be formatted better and more useful. Effectively adds a few new ones as well due to their wording being more general.

Also removes a redundant tip relating to machine upgrading.

## Why / Balance
A lot of these were worded kind oddly, not much to really add. I'm the one who wrote most of these, I'd know.

As for specifics:
The machine upgrading scientist tip now encourages upgrading the lathe since that's probably the most useful application of machine upgrading in the game + the biomass reclaimer is effectively removed.
The uranium spear tip was changed to a more general one since other sources of toxin damage exist; uranium spears are also not versatile or useful enough to warrant a mention.
The hardsuit tip was changed to be a much more useful and general one about speed in combat, and is the main reason I created the PR. The old one was kind of passive-aggressive towards hardsuit mains and misses the main point as to why wearing hardsuits in combat is usually a dumb idea for the user.

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Prototypes/Datasets/tips.yml [hunk 1]
```diff
@@ -30,16 +29,17 @@
   - "Riot armor is significantly more powerful against opponents that aren't using guns compared to regular armor."
   - "As a ghost, you can use the Verb Menu to orbit around and follow any entity in game automatically."
   - "As a Traitor, you may sometimes be assigned to hunt other traitors, and in turn be hunted by others."
-  - "As a Traitor, the syndicate headset can be used to communicate with any other traitors who may have purchased it through a secure channel."
-  - "As a Traitor, never forget the power of the syndicate toolbox. For only 2 telecrystals, you can get a bunch of full set of tools to boost you in an emergency, along with the toolbox itself packing a decent punch."
-  - "As a Traitor, never underestimate the web vest. It may not provide space protection, but its cheap cost and robust protection makes it handy for trigger-happy foes."
+  - "As a Traitor, the syndicate encryption key can be used to communicate with any other traitors who may have purchased it through a secure channel."
```
Comment (Partmedia) [inline Resources/Prototypes/Datasets/tips.yml:32]: It sounds like the encryption key was "purchased through a secure channel", move the clause "through a secure channel" after "communicate"

File: Resources/Prototypes/Datasets/tips.yml [hunk 2]
```diff
@@ -30,16 +29,17 @@
...
+  - "As a Traitor, compromising important communications channels like security or engineering can give invaluable info. Be aware that this goes in both ways - security can compromise syndicate communications as well!"
+  - "As a Traitor, the syndicate toolbox is extremely versatile. For only 2 telecrystals, you can get a full set of tools to boost you in an emergency, insulated combat gloves, and a syndicate gas mask."
```
Comment (Partmedia) [inline Resources/Prototypes/Datasets/tips.yml:34]: I don't really understand what "boost you" means

File: Resources/Prototypes/Datasets/tips.yml [hunk 3]
```diff
@@ -30,16 +29,17 @@
...
   - "As a ghost, you can use the Verb Menu to orbit around and follow any entity in game automatically."
   - "As a Traitor, you may sometimes be assigned to hunt other traitors, and in turn be hunted by others."
-  - "As a Traitor, the syndicate headset can be used to communicate with any other traitors who may have purchased it through a secure channel."
-  - "As a Traitor, never forget the power of the syndicate toolbox. For only 2 telecrystals, you can get a bunch of full set of tools to boost you in an emergency, along with the toolbox itself packing a decent punch."
-  - "As a Traitor, never underestimate the web vest. It may not provide space protection, but its cheap cost and robust protection makes it handy for trigger-happy foes."
+  - "As a Traitor, the syndicate encryption key can be used to communicate with any other traitors who may have purchased it through a secure channel."
+  - "As a Traitor, compromising important communications channels like security or engineering can give invaluable info. Be aware that this goes in both ways - security can compromise syndicate communications as well!"
```
Comment (Partmedia) [inline Resources/Prototypes/Datasets/tips.yml:33]: "invaluable info" sounds weird, try "valuable information" or "valuable intelligence"

File: Resources/Prototypes/Datasets/tips.yml [hunk 4]
```diff
@@ -60,15 +60,15 @@
   - "As a Detective, you can chase criminals more effectively by using fingerprint and fiber data obtained from forensic scans of objects the perpetrator likely interacted with."
   - "As an Atmospheric Technician, your ATMOS holofan projector blocks gases while allowing objects to pass through. With it, you can quickly contain gas spills, fires and hull breaches."
   - "As an Atmospheric Technician, try to resist the temptation of making canister bombs for Nuclear Operatives, unless you're in a last-ditch scenario. They often lead to large amounts of unnessecarry friendly fire!"
-  - "As an Engineer, you can repair windows by using a welding tool on them while not in combat mode."
+  - "As an Engineer, you can repair cracked windows by using a welding tool on them while not in combat mode."
```
Comment (Partmedia) [inline Resources/Prototypes/Datasets/tips.yml:63]: a "lit" welding tool

File: Resources/Prototypes/Datasets/tips.yml [hunk 5]
```diff
@@ -83,27 +83,27 @@
...
   - "Thrown spears deal extra damage! Beware, however, as throwing them too much will end up breaking them."
-  - "Shivs and spears that are made with uranium glass can deal radiation damage, which requires medicine to properly treat."
-  - "You can throw crafted bolas at people to slow them down, letting you follow up on them for an easier kill or get away."
+  - "All forms of toxin damage are fairly difficult to treat, and usually involve the use of chemicals or other inconvienent methods. You can use this to your advantage in combat."
+  - "You can throw crafted bolas at people to slow them down, letting you follow up on them for an easier kill or getaway."
   - "You can put napalm in a backpack water tank to make a flamethrower."
-  - "Most jobs have alternate uinforms in their drobe machine. Don't be afraid to try out a new look!"
-  - "As any other job other than salvage, it is usually wise to avoid using hardsuits unless you need them, since the slowdown they produce is often much more detrimental than any resistances or spaceproofing you may get."
+  - "Some jobs have alternate uinforms in their respective drobe vendors. Don't be afraid to try out a new look!"
```
Comment (Partmedia) [inline Resources/Prototypes/Datasets/tips.yml:94]: spell check "uinforms"

File: Resources/Prototypes/Datasets/tips.yml [hunk 6]
```diff
@@ -83,27 +83,27 @@
...
   - "Just because a job can't be a traitor at the beginning of a round doesn't mean that they'll never be a traitor."
-  - "Syndicate gas masks will block flashes, meaning that Nuclear Operatives and other dubious individuals wearing them can't be flashed."
+  - "Syndicate gas masks will both provide welding protection and block flashes. Think twice before trying to flash a Nuclear Operative!"
   - "Demoman takes skill."
   - "You can spray a fire extinguisher, throw items or fire a gun while floating through space to give yourself a minor boost. Simply fire opposite to where you want to go."
-  - "You can drag other players onto yourself to open the strip menu, letting you remove their equipment or force them to wear something. Note that exosuits or helmets will block your access to the clothing beneath them, and that certain items take longer to strip or put on than others."
+  - "You can drag other players onto yourself to open the strip menu, allowing you to remove their equipment or force them to wear something. Note that exosuits or helmets will block your access to the clothing beneath them, and that certain items take longer to strip or put on than others."
```
Comment (Partmedia) [inline Resources/Prototypes/Datasets/tips.yml:100]: More words is not always better

**Review by Partmedia — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Datasets/tips.yml (hunk 2 above)
Comment (Flareguy) [reply Resources/Prototypes/Datasets/tips.yml:34]: To further you along. I don't like how "help" sounds that much in comparison, but considering your confusion I will definitely change it as these are tips meant to be read by everyone.

File: Resources/Prototypes/Datasets/tips.yml (hunk 6 above)
Comment (Flareguy) [reply Resources/Prototypes/Datasets/tips.yml:100]: I changed this so the the scentence flowed better. "Letting you remove their equipment or force them to wear something" doesn't feel quite gramatically correct, but I may be wrong.

**Review by Partmedia — APPROVED** [review]
I'm satisfied with the revisions, thank you.

**Pull request merged**